src/App.js

```diff
@@ -41,7 +41,7 @@ class App extends Component {
       <BrowserRouter>
       <div className="App">
               
-      <Header placeholder={"Search for location"} showSignup={this.hideComponent}/>
+      <Header placeholder={"Search for housing"} showSignup={this.hideComponent}/>
```

Comment:
I think this should be locations, since the backend call has to use it to query for the location?